sepa_sale_order_autoconfirm/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "SEPA Sale Order AutoConfirm",
+    "summary": "Autoconfirm Sale orders created via ecommerce with SEPA products",
+    "version": "16.0.1.0.0",
+    "category": "Website",
+    "website": "https://github.com/coopiteasy/addons",
+    "author": "Coop IT Easy SC",
+    "license": "AGPL-3",
+    "depends": [
+        "payment_sepa_dd",
+    ],
```

Comment:
```suggestion
        "website_sale",
    ],
```

Reply:
I need the field "allow_sepa_dd_payment", that is not in website_sale